app/views/schemes/show.html.erb

```diff
@@ -10,19 +10,28 @@
 
 <%= render partial: "organisations/headings", locals: { main: @scheme.service_name, sub: nil } %>
 
-<%= render SubNavigationComponent.new(items: scheme_items(request.path, @scheme.id, "Locations")) %>
+<% if FeatureToggle.location_toggle_enabled? %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+<% end %>
+      <%= render SubNavigationComponent.new(items: scheme_items(request.path, @scheme.id, "Locations")) %>
 
-<h2 class="govuk-visually-hidden">Scheme</h2>
+      <h2 class="govuk-visually-hidden">Scheme</h2>
 
-<%= govuk_summary_list do |summary_list| %>
-  <% display_scheme_attributes(@scheme).each do |attr| %>
-    <% next if current_user.data_coordinator? && attr[:name] == ("Housing stock owned by") %>
-    <%= summary_list.row do |row| %>
-      <% row.key { attr[:name].eql?("Registered under Care Standards Act 2000") ? "Registered under Care Standards Act 2000" : attr[:name].to_s.humanize } %>
-      <% row.value { attr[:name].eql?("Status") ? status_tag(attr[:value]) : details_html(attr) } %>
-      <% row.action(text: "Change", href: scheme_edit_name_path(scheme_id: @scheme.id)) if attr[:edit] %>
-    <% end %>
-  <% end %>
+      <%= govuk_summary_list do |summary_list| %>
+        <% display_scheme_attributes(@scheme).each do |attr| %>
+          <% next if current_user.data_coordinator? && attr[:name] == ("Housing stock owned by") %>
+          <%= summary_list.row do |row| %>
+            <% row.key { attr[:name].eql?("Registered under Care Standards Act 2000") ? "Registered under Care Standards Act 2000" : attr[:name].to_s.humanize } %>
+            <% row.value { attr[:name].eql?("Status") ? status_tag(attr[:value]) : details_html(attr) } %>
```

Comment:
```suggestion
            <% row.value { attr[:name].eql?("Status") ? status_tag(attr[:value]) : details_html(attr) } %>
```

This logic belongs in the helper, that's what it's for! Can you adjust `SchemesHelper#display_scheme_attributes` directly to wrap `status_tag` and `details_html` where appropriate. We can also adjust the `Registered under Care Standards Act 2000` field and pass in `current_user` so that we can filter out the `Housing stock owned by` column.